Plotter/python/sample/ResultDict.py

```diff
@@ -364,9 +364,9 @@ def run(self,graphs=True,rdf_dict=None,dot=False,verb=0):
     start = time.time(), time.process_time() # wall-clock & CPU time
     if not results:
       LOG.warn("ResultDict.run: Did not get any results to run... self._dict=%r"%(self._dict))
-    elif graphs: # should be faster for large number of results in parallel
-      LOG.verb("ResultDict.run: Start RunGraphs of %s results with %s threads..."%(len(results),ROOT.GetThreadPoolSize()),verb,1)
-      RDF.RunGraphs(results) # run results concurrently
+      #elif graphs: # should be faster for large number of results in parallel
```

Comment:
Please undo these changes. `RDF.RunGraphs` should be available in the newer CMSSW versions and is 10–30% faster.

Reply:
done